src/search/pdbs/utils.cc

```diff
@@ -67,6 +68,37 @@ vector<int> get_non_goal_variables(const TaskProxy &task_proxy) {
     return non_goal_variables;
 }
 
+vector<vector<int>> compute_cg_neighbors(
```

Comment:
I would expect this to be part of the namespace causal_graph, where I would make it slightly more general by splitting the bool bidirectional to a bool add_predeccessors and a bool add_successors (i.e., also supporting the case where only successors are requested).

Reply:
I'm not so sure. The `CausalGraph` class itself already provides the members `get_successors` and `get_predecessors`. Providing a function to store them doesn't make too much sense to me. This is seems specific to the application we have here, which stores the neighbors to be able to shuffle them.

Reply:
We decided to not change anything here.